Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.h

```diff
@@ -126,6 +130,15 @@ class GDKTESTGYMS_API ABenchmarkGymGameModeBase : public AGameModeBase
 	FMetricTimer RequiredPlayerReportTimer;
 	FMetricTimer RequiredPlayerCheckTimer;
 	FMetricTimer DeploymentValidTimer;
+
+	// For sim player movement metrics
+	float ExpectedAvgVelocity;
```

Comment:
Remove this member because it is not used any more.

Reply:
okay